Cargo.toml

```diff
@@ -45,3 +45,8 @@ incremental = false
 codegen-units = 1
 inherits = "test"
 
+[package.metadata.cross.target.aarch64-unknown-linux-musl]
+dockerfile = "./tools/cross/Cross.Dockerfile.aarch64"
+
+[package.metadata.cross.target.armv7-unknown-linux-musleabihf]
+dockerfile = "./tools/cross/Cross.Dockerfile.armv7"
```

Comment:
`package.metadata.cross.target.armv7-unknown-linux-musleabihf` should be `ockam_command.metadata.cross.target.armv7-unknown-linux-musleabihf`